docs/UserGuide.md

```diff
@@ -330,12 +424,35 @@ GuestBook data is saved in the hard disk **automatically** after any command tha
 GuestBook data is saved as a _JSON file_ `[JAR file location]/data/guestbook.json`.<br>
 Advanced users may change data directly by editing this data file.
 
-<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
-If your changes to the data file makes its format invalid, GuestBook will discard all data and start with an empty data file at the next run.
+<div markdown="span" class="alert alert-warning">
+
+:exclamation: **Caution:**<br>
+* If your changes to the data file makes its format invalid, GuestBook will discard all data and start with an empty data file at the next run.
 </div>
 
 [Return to Table of Contents](#table-of-contents)
 
+--------------------------------------------------------------------------------------------------------------------
+
+### Constraints of fields
+
+The constraints of fields are placed in a separate section as we do not wish to flood you with too much 
+information at once. By following this incremental approach, we hope to make the content more digestible
+for you.
+
+| Field                   | Corresponding prefix | Constraint                                                                                                                                                                                                    |
+|-------------------------|----------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **`NAME`**              | `n/`                 | should only contain alphanumeric characters and spaces.                                                                                                                                                       |
+| **`PHONE`**             | `p/`                 | should only contain numbers, and it should be at least 3 digits and at most 15 digits long.                                                                                                                   |
+| **`EMAIL`**             | `e/`                 | should only contain alphanumeric characters and '@' character. The local segment (before "@") should be at most 64 characters long, and the domain segment (after "@") should be at most 255 characters long. |
```

Comment:
Should we mention that the local segment allows for certain special characters as well but do not allow adjacent ones and cannot start or end with one also? (Basically our message_constraint)

Reply:
This is just for email btw I accidentally selected the other lines.

Reply:
Added.